pages.tr/common/chroot.md

```diff
@@ -0,0 +1,10 @@
+# chroot
+
+> Komut veya etkileşimli komut satırını özel kök diziniyle çalıştırır.
+> Komutu yeni kök dizini olarak çalıştır:
```

Comment:
@navarroaxel you mistook @Yutyo's syntax&mdash;they missed a newline rather than used the wrong punctuation ;)
```suggestion

- Komutu yeni kök dizini olarak çalıştır:
```